src/roulette.py

```diff
@@ -126,3 +126,124 @@ def get(self, bin: int) -> Bin:
         :rtype: Bin
         """
         return self.bins[bin]
+
+
+class BinBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def buildBins(self, wheel: Wheel) -> None:
+        self.create_straight_bets(wheel)
+
+    @staticmethod
+    def create_straight_bets(wheel: Wheel) -> None:
+        odd_for_straight_bet = 35
+        straight_bet_numbers = set(_ for _ in range(37))
+
+        for i in straight_bet_numbers:
+            straight_bet = Outcome(str(i), odd_for_straight_bet)
+            wheel.addOutcome(i, straight_bet)
+        zero_zero_bet = Outcome("00", 35)
```

Comment:
can 35 be replaced with `odd_for_straight_bet`?

Reply:
Done. Thanks.